doc/cephadm/operations.rst

```diff
@@ -278,19 +278,30 @@ You can disable this health warning with::
 /etc/ceph/ceph.conf
 ===================
 
-Cephadm uses a minimized ``ceph.conf`` that only contains 
+Cephadm distributes a minimized ``ceph.conf`` that only contains 
 a minimal set of information to connect to the Ceph cluster.
 
-To update the configuration settings, use::
+To update the configuration settings, instead of manually editing
+the ``ceph.conf`` file, use the config database instead::
 
   ceph config set ...
 
+See :ref:`ceph-conf-database` for details. 
 
-To set up an initial configuration before calling
-`bootstrap`, create an initial ``ceph.conf`` file. For example::
+By default, cephadm does not deploy that minimized ``ceph.conf`` across the
+cluster. To enable the management of ``/etc/ceph/ceph.conf`` files on all
+hosts, please enable this by running::
+
+  ceph config set mgr mgr/cephadm/manage_etc_ceph_ceph_conf true
+
+To set up an initial configuration before bootstrapping 
+the cluster, create an initial ``ceph.conf`` file. For example::
 
   cat <<EOF > /etc/ceph/ceph.conf
   [global]
   osd crush chooseleaf type = 0
   EOF
+
+Then, run bootstrap referencing this file::
+
   cephadm bootstrap -c /root/ceph.conf ...
```

Comment:
unrelated to this PR, it's a little confusing that this line references /root/ceph.conf, while the prior cat stores it in /etc/ceph/ceph.conf